src/index.ts

```diff
@@ -51,61 +57,66 @@ async function getCoreChainID(): Promise<ChainID> {
 
 async function init(): Promise<void> {
   let db: DataStore;
-  switch (process.env['STACKS_BLOCKCHAIN_API_DB']) {
-    case 'memory': {
-      logger.info('using in-memory db');
-      db = new MemoryDataStore();
-      break;
-    }
-    case 'pg':
-    case undefined: {
-      db = await PgDataStore.connect();
-      break;
-    }
-    default: {
-      throw new Error(
-        `Invalid STACKS_BLOCKCHAIN_API_DB option: "${process.env['STACKS_BLOCKCHAIN_API_DB']}"`
-      );
+  const mode = process.env['MODE'];
```

Comment:
The env var should have an application prefix, for example `STACKS_API_x`. Also since this is a boolean, it should probably be something like `STACKS_API_OFFLINE_MODE`, then change the check to:
`if ('STACKS_API_OFFLINE_MODE' in process.env) { // run in offline mode }`